src/mscorlib/src/System/Span.cs

```diff
@@ -74,13 +74,18 @@ private Span(ref T ptr, int length)
 
         public int Length
         {
-            get
-            {
-                return _length;
-            }
+            get { return _length; }
         }
 
-        public static Span<T> Empty { get { return default(Span<T>); } }
+        internal long SizeInBytes
+        {
+            get { return Length * JitHelpers.SizeOf<T>(); }
```

Comment:
One of the multiplicands should be explicitly cast to `nuint`. Otherwise, the multiplication will be done as 32-bit and so it can overflow on 64-bit platforms.